cfgmgr/buffermgr.cpp

```diff
@@ -132,7 +134,13 @@ task_process_status BufferMgr::doSpeedUpdateTask(string port, string speed)
     }
 
     cable = m_cableLenLookup[port];
+    if (cable == "0m")
+    {
+        SWSS_LOG_INFO("Not creating/updating PG profile for port %s. Cable length is set to %s", port.c_str(), cable.c_str());
```

Comment:
Raise this log to NOTICE so that it will by default recorded in syslog?

Reply:
Done!